apps/planning-suite-kit/package.json

```diff
@@ -12,12 +12,12 @@
   },
   "dependencies": {
     "@aragon/apps-finance": "^3.0.0",
-    "@aragon/apps-shared-minime": "^1.0.1",
+    "@aragon/apps-shared-minime": "1.0.1",
     "@aragon/apps-survey": "^1.0.0-rc.2",
     "@aragon/apps-token-manager": "^2.0.0",
     "@aragon/apps-voting": "^2.0.0",
-    "@aragon/apps-vault": "^4.1.0",
-    "@aragon/os": "^4.2.0",
+    "@aragon/apps-vault": "4.1.0",
```

Comment:
I was focusing into Rewards dependencies and pinned the common ones for consistency and to prevent hoisting warnings from lerna, but yeah I agree we should pin every contract dependency to be more in control with the external contracts we use, I can extend the pinning to the rest of apps/kits